### ISSUE TITLE
When going to train mine gets stuck

### ISSUE THREAD

**Issue by Accy587** [SayanoAI/RVC-Studio#12]
Mine just seams like it gets stuck here and never changes no matter how long i wait. it says "INFO:jc_v2_40k:{'train': {'log_interval': 200, 'seed': 1234, 'epochs': 20000, 'learning_rate': 0.0001, 'betas': [0.8, 0.99], 'eps': 1e-09, 'batch_size': 5, 'fp16_run': True, 'lr_decay': 0.999875, 'segment_size': 12800, 'init_lr_ratio': 1, 'warmup_epochs': 0, 'c_mel': 45, 'c_kl': 1.0}, 'data': {'max_wav_value': 32768.0, 'sampling_rate': 40000, 'filter_length': 2048, 'hop_length': 400, 'win_length': 2048, 'n_mel_channels': 125, 'mel_fmin': 0.0, 'mel_fmax': None, 'training_files': './logs\\jc_v2_40k/filelist.txt'}, 'model': {'inter_channels': 192, 'hidden_channels': 192, 'filter_channels': 768, 'n_heads': 2, 'n_layers': 6, 'kernel_size': 3, 'p_dropout': 0, 'resblock': '1', 'resblock_kernel_sizes': [3, 7, 11], 'resblock_dilation_sizes': [[1, 3, 5], [1, 3, 5], [1, 3, 5]], 'upsample_rates': [10, 10, 2, 2], 'upsample_initial_channel': 512, 'upsample_kernel_sizes': [16, 16, 4, 4], 'use_spectral_norm': False, 'gin_channels': 256, 'spk_embed_dim': 109}, 'model_dir': './logs\\jc_v2_40k', 'experiment_dir': './logs\\jc_v2_40k', 'save_every_epoch': 5, 'name': 'jc', 'total_epoch': 1000, 'pretrainG': 'models\\pretrained_v2\\f0G40k.pth', 'pretrainD': 'models\\pretrained_v2\\f0D40k.pth', 'version': 'v2', 'gpus': '0', 'sample_rate': '40k', 'if_f0': 1, 'if_latest': 1, 'save_every_weights': '1', 'if_cache_data_in_gpu': 0}"

And on the home page I already downloaded everything. I did have it train a feature index and it did work.

![opera_Hp4IjJwn2H](https://github.com/SayanoAI/RVC-Studio/assets/99425221/b55a0521-3b94-43bc-843a-ce9f60f7df6c)
![opera_IvbQihwCsY](https://github.com/SayanoAI/RVC-Studio/assets/99425221/1c1e7413-025c-4498-b92d-2826af5b1a4e)
![opera_xCVxhQyZ6L](https://github.com/SayanoAI/RVC-Studio/assets/99425221/df51f078-8a7f-48e5-a6dc-57ee893ec204)

**Comment by SayanoAI** [comment]
why is your "Process Data" button disabled? try replacing the space in your dataset path to "all_rvc/js" and check your ./logs/js folder to see if the training data folders are empty

**Comment by Accy587** [comment]
So i didi remove teh space and renamed the  folder to "all_rvc/js" and the ./logs/js folder dose have training data in it. Still the same thing happens. My gpu is also a 3060 12gb so it should work.

**Comment by SayanoAI** [comment]
did you update to the latest commit?

**Comment by SayanoAI** [comment]
did that help?

**Comment by SayanoAI** [comment]
i can't help you much without knowing the error messages. what does the terminal say after you start training? are you opening the app with conda-start.bat?

**Comment by Accy587** [comment]
yes i am opening it with conda-start.bat and all that the terminal all i says is "INFO:jc_v2_40k:{'train': {'log_interval': 200, 'seed': 1234, 'epochs': 20000, 'learning_rate': 0.0001, 'betas': [0.8, 0.99], 'eps': 1e-09, 'batch_size': 5, 'fp16_run': True, 'lr_decay': 0.999875, 'segment_size': 12800, 'init_lr_ratio': 1, 'warmup_epochs': 0, 'c_mel': 45, 'c_kl': 1.0}, 'data': {'max_wav_value': 32768.0, 'sampling_rate': 40000, 'filter_length': 2048, 'hop_length': 400, 'win_length': 2048, 'n_mel_channels': 125, 'mel_fmin': 0.0, 'mel_fmax': None, 'training_files': './logs\jc_v2_40k/filelist.txt'}, 'model': {'inter_channels': 192, 'hidden_channels': 192, 'filter_channels': 768, 'n_heads': 2, 'n_layers': 6, 'kernel_size': 3, 'p_dropout': 0, 'resblock': '1', 'resblock_kernel_sizes': [3, 7, 11], 'resblock_dilation_sizes': [[1, 3, 5], [1, 3, 5], [1, 3, 5]], 'upsample_rates': [10, 10, 2, 2], 'upsample_initial_channel': 512, 'upsample_kernel_sizes': [16, 16, 4, 4], 'use_spectral_norm': False, 'gin_channels': 256, 'spk_embed_dim': 109}, 'model_dir': './logs\jc_v2_40k', 'experiment_dir': './logs\jc_v2_40k', 'save_every_epoch': 5, 'name': 'jc', 'total_epoch': 1000, 'pretrainG': 'models\pretrained_v2\f0G40k.pth', 'pretrainD': 'models\pretrained_v2\f0D40k.pth', 'version': 'v2', 'gpus': '0', 'sample_rate': '40k', 'if_f0': 1, 'if_latest': 1, 'save_every_weights': '1', 'if_cache_data_in_gpu': 0}" basically and then just freezes.

**Comment by Accy587** [comment]
[train.log](https://github.com/SayanoAI/RVC-Studio/files/12754848/train.log)
all that the train log has in it too no matter how long i let it run

**Comment by SayanoAI** [comment]
does inference and chat work for you?

**Comment by Accy587** [comment]
Inference works but when chating in the terminal it just says press any key to continue when i click start chatting.

**Comment by SayanoAI** [comment]
how many rows of does your "/logs\jc_v2_40k/filelist.txt" have? Does that correspond to the number of datafiles you have?

can you also tell me what OS you're using?

**Comment by Accy587** [comment]
My filelist.txt has 186 lines and i have 46 data set files. I am also using Windows 10 64bit.

**Comment by SayanoAI** [comment]
ok, try pulling the most recent branch and redo the data processing (after deleting your log folder).
just to confirm, are your datafiles in wav format or something else? soundfile has some problems with ogg files so you might run into some issues relating to that.

**Comment by Accy587** [comment]
Yes my dataset is in .wav. And I did do what you said and deleted the logs folder and update. After i did that i tried again and after the loading bar at the bottom hit 100 and i watied for awhile same thing happened.

**Comment by SayanoAI** [comment]
which one of these logs (underlined in red) is missing in your terminal? 
![image](https://github.com/SayanoAI/RVC-Studio/assets/28544690/818af94d-cff7-47b7-bb35-7d35415602ce)

**Comment by Accy587** [comment]
the furthest that mine gets to is line 145: (0, 1, '0')
![cmd_5rjKJFRgS6](https://github.com/SayanoAI/RVC-Studio/assets/99425221/f7509387-7c5b-4daf-9f95-d616b596ae26)

**Comment by SayanoAI** [comment]
looks like your training failed at spawning distributed gpu training. no idea why it's hanging on your computer. could be a driver issue
